pkg/controller/install/rule_checker.go

```diff
@@ -155,3 +163,16 @@ func (c *CSVRuleChecker) hasOwnerConflicts(ownerRefs []metav1.OwnerReference) bo
 
 	return conflicts
 }
+
+func ruleValid(rule rbacv1.PolicyRule) error {
+	if len(rule.Verbs) == 0 {
```

Comment:
does the particular openapi flavor kube supports let you specify `minItems` length on the verb list?